src/time_tracker/invoices/handlers/spec.clj

```diff
@@ -45,6 +45,5 @@
 (s/def ::timers (timers-spec/normalized-timers-spec 0))
 
 (s/def ::invoice-data
-  (s/and (s/merge ::generate-invoice-params-coerced
-                  (s/keys :req-un [::users-spec/users ::timers-spec/timers]))
-         invoice-data-pred))
```

Comment:
All right. Since we want to get rid of this, the predicate at https://github.com/nilenso/time-tracker/pull/83/files#diff-9960a0f13bfce1a10c44230260f8a27eR38 should be deleted since it is dead code.